src/OpenConext/UserLifecycle/Application/CommandHandler/RemoveFromLastLoginCommandHandler.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * Copyright 2018 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace OpenConext\UserLifecycle\Application\CommandHandler;
+
+use OpenConext\UserLifecycle\Application\Command\CommandInterface;
+use OpenConext\UserLifecycle\Application\Command\RemoveFromLastLoginCommand;
+use OpenConext\UserLifecycle\Domain\Repository\LastLoginRepositoryInterface;
+
+class RemoveFromLastLoginCommandHandler implements CommandHandlerInterface
+{
+    /**
+     * @var LastLoginRepositoryInterface
+     */
+    private $lastLoginRepository;
+
+    public function __construct(LastLoginRepositoryInterface $lastLoginRepository)
+    {
+        $this->lastLoginRepository = $lastLoginRepository;
+    }
+
+    /**
+     * @param CommandInterface|RemoveFromLastLoginCommand $command
+     */
+    public function handle(CommandInterface $command)
+    {
+        $this->lastLoginRepository->delete($command->getCollabPersonId()->getCollabPersonId());
```

Comment:
I think `getUser()->getUser()` and similar constructs look funny, would prefer something like `getCollabPersonId` (returns the value object) and `getValue`.

Reply:
I've added a __toString solution as I think it's also funny to implement a getValue'ish method on value objects.